doc/classes/JSONParseResult.xml

```diff
@@ -68,12 +70,24 @@
 	</methods>
 	<members>
 		<member name="error" type="int" setter="set_error" getter="get_error" enum="Error">
+			Returns error's type if JSON source not successfull parsed. See [@Global Scope].Error.
 		</member>
 		<member name="error_line" type="int" setter="set_error_line" getter="get_error_line">
+			Returns error's line if JSON source not successful parsed.
```

Comment:
The line number where the error occurred if JSON source was not successfully parsed.